opennms-doc/guide-admin/src/asciidoc/text/operation/jmx-config-generator/webui.adoc

```diff
@@ -0,0 +1,80 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../../images
+
+=== Web based JMX Configuration Generator
+
+Generating complex _JMX_ data collection configurations can be generated from the a web based tool.
```

Comment:
`the a web based tool` should either be `a web based tool` or `the web based tool`